tests/test_data/entities/nAm3.csv

```diff
@@ -1,6 +0,0 @@
-amy
```

Comment:
Please leave this file in entities for testing. It should serve as test it name and value of entity can contain numbers etc. (should be extended)